pallets/liquidity-pools-gateway/axelar-gateway-precompile/src/lib.rs

```diff
@@ -246,30 +245,16 @@ where
 	#[precompile::public("execute(bytes32,string,string,bytes)")]
 	fn execute(
 		handle: &mut impl PrecompileHandle,
-		command_id: H256,
```

Comment:
Should we drop this or keep as is to keep it consistent with the gateway contract that we're trying to mimic?

Reply:
No! We have to keep it consistent to be easily callable by the EVM side.

Reply:
OK, thanks for clearing that out!